pandas/tests/io/parser/test_parsers.py

```diff
@@ -99,3 +102,14 @@ def read_table(self, *args, **kwds):
         kwds = kwds.copy()
         kwds['engine'] = self.engine
         return read_table(*args, **kwds)
+
+
+class TestParameterValidation(object):
```

Comment:
Let's move this class into `test_common.py` and use `self.read_csv` instead of `read_csv`.  This base test class should not be touched for organizational purposes.